oat-sa/tao-test-runner-qti-fe
# Fix/tca 439/menu announce multiple times

**Related to**
https://oat-sa.atlassian.net/browse/TCA-439

**Description**
Selected option and menu title is announced multiple times for Tools and Contrast

**How to test**
1. As a test-taker start a test
2. Navigate to the bottom bar, set focus on "Contrast" button
3. Press Space / Enter to expand the list of options
4. JAWS should announce just "listbox - Black on white checked ..." without repetatives
